test/parallel/test-cluster-shared-handle-bind-reuseaddr.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+var cluster = require('cluster');
+var dgram = require('dgram');
+var spawn = require('child_process').spawn;
+
+if (common.isWindows) {
+  console.log('1..0 # Skipped: `reuseAddr: true` not supported on Windows');
```

Comment:
Is this true?  Libuv certainly understands the UV_UDP_REUSEADDR flag on Windows.